MKOB/kobmain.py

```diff
@@ -136,12 +138,26 @@ def update_sender(id):
 
 def readerCallback(char, spacing):
     """display characters returned from the decoder"""
-    n = int(spacing + 0.5)
-    if n > 5:
-        sp = "__" if char != "~" else ""
+    if kc.CodeType == 0:  # TODO: use config enum for American
```

Comment:
I believe that change was merged; please use "CodeType.american" here now?

Reply:
I totally agree. I was aware that the new config had been merged on the server, but my branch was based on an earlier version of Master and the new config changes weren't reflected in my working directory. I'm sure there's a way I could have merged the new config into my branch, but I wasn't sure how to do it.

I will definitely change to CodeType.american the next time I touch the code (if someone else doesn't get to it first). I know that needs to be done before the enum values get changed, per your earlier suggestion.

Reply:
I'll pull this branch, merge, make this change, and push it back.